src/MicrosoftTeams.ts

```diff
@@ -760,12 +793,20 @@ namespace microsoftTeams {
          * Notifies the frame that initiated this authentication request that the request failed.
          * This function is usable only on the authentication window.
          * This call causes the authentication window to be closed.
-         * @param reason Specifies a reason for the authentication failure. If specified, the frame that initiated the authentication pop-up receives this value in its callback.
+         * @param authenticationResultParams Specifies a result for the authentication. It contains the optional values of state(URL to redirect back after authentication in Outlook Desktop), reason(in case of failure), result(in case of success).
          */
-        export function notifyFailure(reason?: string): void {
+        export function notifyFailure(authenticationResultParams: AuthenticationResultParameters): void {
+            if (authenticationResultParams.state) {
```

Comment:
I don't see a unit test for this code path (i.e. the one where auth happens via redirect). Consider adding one.

Reply:
added unit tests